test/KernelInfo.rb

```diff
@@ -0,0 +1,67 @@
+# !/usr/bin/env ruby -w
+
+require 'rmagick'
+require 'test/unit'
+require 'test/unit/ui/console/testrunner' unless RUBY_VERSION[/^1\.9|^2/]
+
+class KernelInfoUT < Test::Unit::TestCase
+  setup do
+    @kernel = Magick::KernelInfo.new('Octagon')
+  end
+
+  def test_new
+    Magick::KernelInfoType.values do |kernel|
+      k = kernel.to_s.sub('Kernel', '')
+
+      if kernel == Magick::UserDefinedKernel
+        assert_raise(RuntimeError) { Magick::KernelInfo.new(k) }
+      else
+        assert_instance_of(Magick::KernelInfo, Magick::KernelInfo.new(k))
+      end
+    end
+    assert_raise(RuntimeError) { Magick::KernelInfo.new('') }
+    assert_raise(TypeError) { Magick::KernelInfo.new(42) }
+  end
+
+  def test_zero_nans
```

Comment:
Maybe remove this tests? It fails with ruby 2.3 on Windows and this method is deprecated and will be removed soon.

Reply:
Looks like it's causing a seg fault.

Reply:
@dlemstra NOT agreed.
When we will totally stop to support them, then we will just remove unnecessary tests.

Reply:
I just think that you should not spend to much of your time on this. It is very unlikely that someone is using this method.

Reply:
It's not a job, so I think we do whatever you like.  Now I want to improve test for the future ;)